File: modul4/Python/PRAK403_2310817210017_MUHAMMADNURHUDABAIHAQI.py
while True:
    num1, num2 = map(int, input("").split())

    if num1 == 0 and num2 == 0:
        break

    print(f"{num1} {num2}", end='')

    step = -1 if num1 > num2 else 1
    for i in range(num1 + step, num2, step):
        print(f" - {i} {num1 + num2 - i}", end='')

    print(f" - {num2} {num1 + num2 - num2}")

File: modul4/C/PRAK403_2310817210017_MUHAMMADNURHUDABAIHAQI.c
#include <stdio.h>

int main() {
    int num1, num2;

    while (1) {
        printf("");
        scanf("%d %d", &num1, &num2);

        if (num1 == 0 && num2 == 0) {
            break;
        }

        printf("%d %d", num1, num2);

        int step = (num1 > num2) ? -1 : 1;
        for (int i = num1 + step; i != num2; i += step) {
            printf(" - %d %d", i, num1 + num2 - i);
        }

        printf(" - %d %d\n", num2, num1 + num2 - num2);
    }

    return 0;
}
